### ISSUE TITLE
BCH 

### ISSUE THREAD

**Issue by kniraj** [bitpay/copay-recovery#83]
I had around 0.199164 BTC in my copay wallet on the 29th July 2017. I transferred these on the 13th Aug to my exchange.
Now when I try to recover the BCH for these (using the recovery tool), I constantly get the message "Available balance: 0.00000000 BCH. Insufficents funds."

Please assist. How can I recover my BCH

**Comment by xanatas** [comment]
I have the same thing did not touch the copay funds since ever. its an old wallet hence the in-build function in the mobile wallet is not working. its a multisig wallet as well but it shows no funds on BCH and on BCT. any recommendations?

**Comment by JDonadio** [comment]
Please, go to settings > bitcoin cash support and scan your wallets

<img width="372" alt="copay_-_copay_bitcoin_wallet" src="https://user-images.githubusercontent.com/13851807/31389509-9493e838-ada7-11e7-9513-79c19de01693.png">

Check if your wallets appear listed as available to duplicate

**Comment by xanatas** [comment]
The wallet is too old it says not supported

**Comment by JDonadio** [comment]
If that's the case, Copay/BitPay doesn't support the duplicate to BCH process for old wallets. Instead, please use our [recovery tool](https://bitpay.github.io/copay-recovery/) to scan your BCH and send the balance to a new fresh BCH wallet.

Note: ensure to select `bch/livenet` as chain to scan your BCH balance from your BTC wallet.

Hope that helps

**Comment by xanatas** [comment]
I No it doesnt, as kniraj  stated in top comment this is what shows up after scan: "Available balance: 0.00000000 BCH. Insufficents funds." also if when I scan for btc/livenet

**Comment by JDonadio** [comment]
Did you create your wallet from Copay/BitPay app? or did you import it from an external seed?

**Comment by xanatas** [comment]
no the wallet was created with the copay app. multisig 3 out of 6 BIP45.

**Comment by matiu** [comment]
@kniraj at what address do you have your BCH? Can you check that address on a bitcoin cash block explorer?  Like this one https://cashexplorer.bitcoin.com/

**Comment by xanatas** [comment]
Thanks for the tool, i checked all my addresses, all Bitcoin Cash are there.

**Comment by kniraj** [comment]
Hi, Thanks for responses, however no luck
1. I tried to use the wallet on my mobile to scan, but it did not return any result
2. On the Cash Explorer what address should I enter ? Is it the one which is shown on the wallet (like a hash string?)

**Comment by xanatas** [comment]
Yes its your Bitcoin address where you think it has Bitcoin Cash funds in it.

**Comment by xanatas** [comment]
Is there anything that I can check in the settings to debug the process or see a verbose output?
I have a lot of Bitcoin Cash stuck, Do you know another tool that can deal with Multisig cash sweeping?

**Comment by gabrielbazan7** [comment]
Hi ! we found the issue today. We are working on it. We will let you know when the new version is published. Sorry about that.

**Comment by kniraj** [comment]
That's a great news. Looking forward to it. Will this be a new version of the app or the recovery tool ?

**Comment by gabrielbazan7** [comment]
A new version of the recovery tool. And, It's already published !

**Comment by kniraj** [comment]
Hi, I tried to search again today but I get the same message.

**Comment by xanatas** [comment]
Same here, no funds found.

**Comment by morrisonbrett** [comment]
Same here. I have older (Non BIP44) wallet. Recovery tool yields nothing. Also tried many steps here: https://bip32jp.github.io/english/

**Comment by gabrielbazan7** [comment]
@morrisonbrett  1 - 1 wallet ?

**Comment by morrisonbrett** [comment]
![image](https://user-images.githubusercontent.com/1159718/32694800-96324ef8-c6fe-11e7-917f-0fd6f4f58e9e.png)

**Comment by morrisonbrett** [comment]
![image](https://user-images.githubusercontent.com/1159718/32694807-d6eaf986-c6fe-11e7-8134-202d82ed11ed.png)

**Comment by morrisonbrett** [comment]
@gabrielbazan7 are my coins lost?

**Comment by bitbillion** [comment]
Hai, anybody here can help me with my copay wallet problem.

My copay wallet btc is appear with zero balance when i make the duplicate to bch. 

In bch wallet "scanning fund" (Please wait) until now nothing amount are showing. In my btc wallet balance is 0.00btc how can i get back my btc ???!

Please help me.

**Comment by bitbillion** [comment]
Thanks god! Its just appeared :D

**Comment by gabrielbazan7** [comment]
Closing this one, in order to https://github.com/bitpay/copay-recovery/issues/89

**Comment by kniraj** [comment]
Hi I am still not able to retrieve the BCH is there some update on this
issue ?

Regards
Niraj
